vue-frontend/src/components/partials/NewHeader.vue

```diff
@@ -68,28 +68,33 @@
                 </router-link>
                 <a
                   v-else
-                  class="tw-inline-block tw-relative tw-mr-[20px] sm:tw-mr-[30px] lg:tw-mr-[8px] xl:tw-mr-[30px] after:tw-absolute after:tw-w-full after:tw-h-[3px] after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent"
+                  class="tw-relative tw-mr-[20px] tw-inline-block after:tw-absolute after:tw-bottom-0 after:tw-left-0 after:tw-top-[27px] after:tw-h-[3px] after:tw-w-full after:tw-origin-bottom-left after:tw-scale-x-0 after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-duration-300 after:tw-bg-gradient-[90deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-bg-clip-text hover:tw-text-transparent hover:tw-bg-gradient-[270.11deg] hover:after:tw-origin-bottom-left hover:after:tw-scale-x-100 sm:tw-mr-[30px] lg:tw-mr-[8px] xl:tw-mr-[30px]"
                   :href="navbar.url"
                   :target="navbar.target"
                   >{{ navbar.name }}</a
                 >
+
                 <ul
                   v-show="showContributionMenu"
                   @mouseleave="showContributionMenu = false"
-                  class="tw-absolute xl:tw-left-[296px] 2xl:tw-left-[475px] lg:tw-top-[4.5rem] tw-w-max tw-border tw-rounded-[5px] tw-bg-white tw-shadow-md tw-px-6 tw-py-[1.5rem] tw-flex-col tw-space-y-6"
-                  :class="
+                  class="tw-absolute tw-w-max tw-flex-col tw-space-y-6 tw-rounded-[5px] tw-border tw-bg-white !tw-px-6 tw-py-[1.5rem] tw-shadow-md lg:tw-top-[4.5rem] xl:tw-left-[296px] 2xl:tw-left-[475px]"
+                  :class="[
                     subMenus.length == 2
                       ? 'lg:tw-left-[180px]'
-                      : 'lg:tw-left-[169px]'
-                  "
+                      : 'lg:tw-left-[169px]',
+                    currentRoutePath.includes('jobs/') &&
```

Comment:
I think we can fix it on the `jobs/:id` page instead of here. The menu is not the same as other pages on this page still